var/spack/repos/builtin/packages/verrou/package.py

```diff
@@ -0,0 +1,107 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import glob
+import os
+import sys
+
+
+class Verrou(AutotoolsPackage):
+    """A floating-point error checker.
+
+    Verrou helps you look for floating-point round-off errors in programs. It
+    implements a stochastic floating-point arithmetic based on random rounding:
+    all floating-point operations are perturbed by randomly switching rounding
+    modes. This can be seen as an asynchronous variant of the CESTAC method, or
+    a subset of Monte Carlo Arithmetic, performing only output randomization
+    through random rounding.
+    """
+
+    homepage = "https://github.com/edf-hpc/verrou"
+    url      = "https://github.com/edf-hpc/verrou/archive/v2.0.0.tar.gz"
+
+    version('develop',
+            git='https://github.com/edf-hpc/verrou.git',
+            branch='master')
+
+    version('2.0.0', '388d493df3f253c9b049ce0ceae55fd6')
+    version('1.1.0', '9752d776fb534890e5e29f9721ee6125')
+
+    resource(name='valgrind-3.13.0',
+             url='https://sourceware.org/pub/valgrind/valgrind-3.13.0.tar.bz2',
+             sha256='d76680ef03f00cd5e970bbdcd4e57fb1f6df7d2e2c071635ef2be74790190c3b',
+             when='@1.1.0:,develop')
```

Comment:
The `,develop` isn't actually needed anymore, develop sorts higher than any other version.

Reply:
Indeed, this is a remnant from the time where the development version was called "master". Fixed!